### PR TITLE
Products

### PR DESCRIPTION
**Pull request by OleksandrPozden**
List of products with the following features:
- pagination
- permissions
- filtering
- ordering

### CODE REVIEW

**Comment by OleksandrPozden** [comment]
@yura0611 @Orest2205 Look! Now the commits are verified!
image

File: backend/products/models.py [hunk 1]
```diff
@@ -2,13 +2,38 @@
...
+        primary_key=True,
+        related_name="inventory",
+    )
+    quantity = IntegerField(default=0, null=False, validators=[MinValueValidator(0)])
+    sold_qty = IntegerField(default=0, null=False, validators=[MinValueValidator(0)])
+    created_at = DateTimeField(auto_now_add=True)
+    updated_at = DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = "products_product_inventory"
```
Comment (OleksandrPozden) [inline backend/products/models.py:39]: need representation for ProductInventory table

File: backend/products/views.py [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
 class ProductViewSet(ModelViewSet):
-    queryset = Product.objects.all()
+    queryset = Product.objects.select_related('inventory')
     serializer_class = ProductSerializer
+    filterset_class = ProductFilter
 
+    def get_permissions(self):
+        if self.action not in ('list', 'detail'):
+            return [IsAdminUser()]
+        return super().get_permissions()
```
Comment (OleksandrPozden) [inline backend/products/views.py:16]: newline in the end, please

**Comment by yura0611** [comment]
> @yura0611 @Orest2205 Look! Now the commits are verified! image

That very cool

**Review by yura0611 — APPROVED** [review]
Looks pretty good

**Pull request merged**